webgl/penworld.js

```diff
@@ -210,6 +210,11 @@ class Subchunk {
       block.pos = pos
     }
     this.updateBlockFaces(pos)
+    
+    // Ensures that if a newly deleted block has no neighbors, facesChanged will be set to true.
```

Comment:
fixes bug where newly deleted blocks without neighbors still render

example:
![image](https://github.com/SheepTester/platformre/assets/114318049/a0b3dbd9-61cf-4304-b0fd-07b803f3cfb7)

after block is placed and faces are updated:
![image](https://github.com/SheepTester/platformre/assets/114318049/2bafd898-789e-4de8-9a4d-dd1b4ed85398)

the issue was that when no faces are modified, `facesChanged` remains `false`; but when u delete a block with no neighbors, it deletes the block without setting it to `true`. usually if a deleted block has neighbors, their faces will be updated and it will be set to `true` anyways. in this case, if it has no neighbors, it won't be set to true and the faces won't be updated.